apps/rush-lib/src/logic/selectors/GitChangedProjectSelectorParser.ts

```diff
@@ -14,14 +14,22 @@ export class GitChangedProjectSelectorParser implements ISelectorParser<RushConf
   public async evaluateSelectorAsync(
     unscopedSelector: string,
     terminal: ITerminal,
-    parameterName: string
+    parameterName: string,
+    forIncrementalBuild: boolean
   ): Promise<Iterable<RushConfigurationProject>> {
```

Comment:
This function signature isn't super intuitive.  Could we add some docs for what "unscopedSelector" and "parameterName" do, with example strings for each one?

The `forIncrementalBuild` boolean is also not super readable (what does `, false)` mean)?  I'd suggest that by either:

1. Converting all these into an `IEvaluateSelectorOptions` interface so they're passed by name, OR
2. Instead of a boolean, use a type like `'RushChange'|'IncrementalBuild'` or an enum like `EvaluateSelectorMode.RushChange`, OR
3. Maybe eliminate this mixed behavior entirely, and split it into two functions, or inline the code (which is basically 2 statements depending on your IF branch)